### PR TITLE
Move timezone to timezone.properties file. Refactor booking status when created

### PR DESCRIPTION
**Pull request by olexandra-shokotko**

### CODE REVIEW

**Comment by CalculusEnjoyer** [comment]
|File|Coverage [68.06%]|:green_apple:|
|:-|:-:|:-:|
|[WeekService.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/3b727292ffe03fb2ada882e5e404e0e2ce00fa0d/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FWeekService.java)|100%|:green_apple:|
|[BookingDto.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/3b727292ffe03fb2ada882e5e404e0e2ce00fa0d/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2Fdto%2FBookingDto.java)|82.61%|:green_apple:|
|[InterviewPlanningApplication.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/3b727292ffe03fb2ada882e5e404e0e2ce00fa0d/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2FInterviewPlanningApplication.java)|79.17%|:green_apple:|
|[CoordinatorController.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/3b727292ffe03fb2ada882e5e404e0e2ce00fa0d/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FCoordinatorController.java)|54.55%|:green_apple:|

|Total Project Coverage|59.97%|:green_apple:|
|:-|:-:|:-:|

File: src/main/java/com/intellias/intellistart/interviewplanning/services/WeekService.java [hunk 1]
```diff
@@ -6,15 +6,26 @@
...
 /**
  * Week service.
  */
 @Service
+@Component
 public class WeekService {
 
-  public static final ZoneId ZONE_ID = ZoneId.of("Europe/Kiev");
+  @Value("${current.timezone}")
+  public String zone;
```
Comment (VladOSIO2) [inline src/main/java/com/intellias/intellistart/interviewplanning/services/WeekService.java:21]: is this non-static String literal is essential?

Comment (olexandra-shokotko) [reply src/main/java/com/intellias/intellistart/interviewplanning/services/WeekService.java:21]: Yes. It is necessary to put the value from the properties file into a variable using the annotation @Value. String of zone cannot be placed directly into a method ZoneId.of()

**Comment by CalculusEnjoyer** [comment]
|File|Coverage [68.06%]|:green_apple:|
|:-|:-:|:-:|
|[WeekService.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/ffb3ed9350437a593890a8f4a05709c8f1f47e2e/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservices%2FWeekService.java)|100%|:green_apple:|
|[BookingDto.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/ffb3ed9350437a593890a8f4a05709c8f1f47e2e/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2Fdto%2FBookingDto.java)|82.61%|:green_apple:|
|[InterviewPlanningApplication.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/ffb3ed9350437a593890a8f4a05709c8f1f47e2e/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2FInterviewPlanningApplication.java)|79.17%|:green_apple:|
|[CoordinatorController.java](https://github.com/CalculusEnjoyer/intellistart-java-2022-vanquishers/blob/ffb3ed9350437a593890a8f4a05709c8f1f47e2e/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontrollers%2FCoordinatorController.java)|54.55%|:green_apple:|

|Total Project Coverage|59.97%|:green_apple:|
|:-|:-:|:-:|

File: src/main/java/com/intellias/intellistart/interviewplanning/services/WeekService.java (hunk 1 above)
Comment (VladOSIO2) [reply src/main/java/com/intellias/intellistart/interviewplanning/services/WeekService.java:21]: Understood. Thanks for the explanation!

**Review by VladOSIO2 — APPROVED** [review]
Good, Approved

**Pull request merged**